model_bakery/baker.py

```diff
@@ -68,7 +69,18 @@ def make(
     if _valid_quantity(_quantity):
         raise InvalidQuantityException
 
-    if _quantity:
+    if _quantity and _bulk_create:
+        return baker.model._base_manager.bulk_create(
+            [
+                baker.prepare(
+                    _save_kwargs=_save_kwargs,
+                    _refresh_after_create=_refresh_after_create,
```

Comment:
I think we don't need this param here.

Reply:
Good catch. I naively copied the arguments from the `make` call. I will remove that argument.